modules/ROOT/pages/secret-group-add-tls-context.adoc

```diff
@@ -0,0 +1,115 @@
+= TLS Context
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After you create a secret group, you can add a Transport Layer Security (TLS) context to encrypt inbound traffic to Anypoint Runtime Fabric (Runtime Fabric) or Mule 4 proxies. 
+
+You can use the TLS context you store in the secrets manager either when you configure an HTTPS API proxy, or to validate the SSL handshake in the edge endpoint of Runtime Fabric. When you add the TLS context, you can select the ciphers to use with the TLS context, and enable client authentication for Anypoint Security targets. 
+
+== Add TLS Context 
+
+Adding a TLS context to your secret group involves supplying a name, target, security information, version, keystore and, optionally, truststore. 
+
+. In the *Secret Groups* list view, select the secret group to which to add a TLS context. 
+. Select *TLS Context* in the menu on the left, and click *Add TLS Context*. 
+. In the Create TLS Context screen, add the required information:
++
+* *Name* +
+Enter a name for your TLS context. 
+* *Target* +
+Select the target for the TLS context from the drop-down list:
+** *Anypoint Security* +
+Validates the SSL handshake for Runtime Fabric. +
+If you choose *Anypoint Security*, you can select keystores and truststores that are either PEM or non-PEM types. Supported types include: +
+*** JKS
+*** PKCS12
+*** JCEKS 
+** *Mule* +
+Uses the TLS context as the SSL validation for Mule 4-based API proxies.
+* *TLS Version* +
+Select from the following supported TLS versions: 
++
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Target |Supported TLS Version
+|Mule a| * TLS 1.1
+* TLS 1.2
+
+|Anypoint Security a| * TLS 1.1
+* TLS 1.2
+* TLS 1.3
+|===    
+* *Keystore* +
+From the drop-down list, select the keystore to store the TLS context.
+* *Truststore* +
+Optionally select a truststore to which to add the TLS context if you are using a truststore to store certificates trusted by the client. +
+If the Target value for the TLS context is *Mule*, you can select *Insecure* if you don't want certificate validation enforced. 
+* *Expiration Date* +
+Optionally select the expiration date for the certificate.
+. Click *Save*.
+
+Additionally, you can enable client authentication and select the ciphers to use, or create custom ciphers.
+
+=== Enable Client Authentication
+
+If you choose a Target value of *Anypoint Security* when adding your TLS context, you can optionally enable client authentication. TLS-enabled clients verify that they trust the server by validating the public certificate against their own set of trusted certificates.
```

Comment:
```suggestion
If you choose a target value of *Anypoint Security* when adding your TLS context, you can optionally enable client authentication. TLS-enabled clients verify that they trust the server by validating the public certificate against their own set of trusted certificates.
```
Can we replace "optionally" with an "if" statement that explains under what circumstances the audience wants or needs to employ the "optional" action?